### DISCUSSION TITLE
深圳地铁的更名史还有一段

### DISCUSSION THREAD

**Discussion by Mick666-qq** [Ivysauro/CNRT#39] [Ideas]
例如1号线就有1号线罗宝线的名称

**Comment by Ivysauro** [comment]
好的 核实后添加

**Comment by chenw23** [comment]
复核确认：所述深圳地铁1号线历史已经在当前 `data/SpSta.md` 中记录为“1号线→罗宝线→1号线（2008.4，2013.10）”，因此无需重复修改。
